packages/api/endpoints/executions.js

```diff
@@ -31,18 +36,20 @@ async function list(req, res) {
  */
 async function get(req, res) {
   const arn = req.params.arn;
-
-  const e = new models.Execution();
-
+  const knex = await getKnexClient({ env: process.env });
+  const executionPgModel = new ExecutionPgModel();
+  let executionRecord;
   try {
-    const response = await e.get({ arn });
-    return res.send(response);
+    executionRecord = await executionPgModel.get(knex, { arn });
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
       return res.boom.notFound(`No record found for ${arn}`);
```

Comment:
Same as above, we could just return the message from `RecordDoesNotExist`. Super minor though. Either makes sense.

Reply:
e7ff2de35

Reply:
```
if (error instanceof RecordDoesNotExist) {      
  return res.boom.notFound(`Execution ${error.message}`);
}
```
But I don't feel super strongly about it 😄